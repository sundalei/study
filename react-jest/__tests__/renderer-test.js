import renderer from 'react-test-renderer';
import React from 'react';
import { Link } from 'react-router';

test('Link renders correctly', () => {
    const tree = renderer.create(
        <Link page="http://www.baidu.com">Baidu</Link>
    ).toJSON();

    expect(tree).toMatchSnapshot();
});