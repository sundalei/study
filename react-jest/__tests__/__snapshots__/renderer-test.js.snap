exports[`test Link renders correctly 1`] = `
<a
  onClick={[Function]}
  page="http://www.baidu.com"
  style={Object {}}>
  Baidu
</a>
`;
